go/vt/vttablet/tabletserver/repltracker/repltracker.go

```diff
@@ -47,6 +48,8 @@ var (
 	heartbeatLagNsHistogram = stats.NewGenericHistogram("HeartbeatLagNsHistogram",
 		"Histogram of lag values in nanoseconds", []int64{0, 1e6, 1e7, 1e8, 1e9, 1e10, 1e11, 1e12},
 		[]string{"0", "1ms", "10ms", "100ms", "1s", "10s", "100s", "1000s", ">1000s"}, "Count", "Total")
+
+	errFallback = errors.New("heartbeat")
```

Comment:
@ejortegau could we make this error more descriptive?

Reply:
My cat ate my error message 🐱 

Will fix.